demo/src/main.ts

```diff
@@ -1,6 +1,6 @@
 import { enableProdMode } from '@angular/core';
 import { bootstrap } from '@angular/platform-browser-dynamic';
-import { ELEMENT_PROBE_PROVIDERS } from '@angular/platform-browser';
+import { ELEMENT_PROBE_PROVIDERS } from '@angular/platform-browser/src/dom/debug/ng_probe';
```

Comment:
No clue - @Foxandxss, do you know the reason? Removing the logic around here doesn't seem to break anything.